autoload/flog/state.vim

```diff
@@ -161,11 +162,15 @@ function! flog#state#GetCommitRefs(commit) abort
     " full: Full path including refs/.*/
     " path: Path with remote
     " tail: End of path only (no remote)
+    let l:remote = l:match[8][ : -2]
```

Comment:
This block could be moved above the comment which should be above the return value.